### ISSUE TITLE
Key Authentication

### ISSUE THREAD

**Issue by socalengineer** [dstuecken/sshfs-gui#20]
Can you add public/private key authentication?

**Comment by dstuecken** [comment]
I agree we should add this as an additional field where you can browse the keyfile, but in the meantime you can achieve this by adding this to the command line options field:

```-o IdentityFile=/path/to/key```

**Comment by zadorozhko** [comment]
`IdentityFile` option missing in my sshfs.
SSHFS 3.7.3, FUSE library 3.17.2, FUSE kernel interface 7.40, macFUSE mount 5.0.5
I used 
`-o ssh_command='ssh -i /path/to/key.cer'`
and this works.
